### PR TITLE
ac: add pt_BR translation

### PR DESCRIPTION
**Pull request by HTsuyoshi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/ac.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ac
+
+> Imprime estatisticas do tempo usuários conectados.
+> Mais informações: <https://man.openbsd.org/ac>.
+
+- Imprimir a quantidade de tempo que o usuário atual está conectado em horas:
```
Comment (navarroaxel) [inline pages.pt_BR/common/ac.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Imprime a quantidade de tempo que o usuário atual está conectado em horas:
```

File: pages.pt_BR/common/ac.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`ac`
+
+- Imprimir a quantidade de tempo que os usuários estão conectado em horas:
```
Comment (navarroaxel) [inline pages.pt_BR/common/ac.md:10]:
```suggestion
- Imprime a quantidade de tempo que os usuários estão conectado em horas:
```

File: pages.pt_BR/common/ac.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`ac -p`
+
+- Imprimir a quantidade de tempo que um usuário específico está conectado em horas:
+
+`ac -p {{nome_do_usuario}}`
+
+- Imprimir a quantidade de tempo que um usuário específico está conectado em horas por dia (com total):
```
Comment (navarroaxel) [inline pages.pt_BR/common/ac.md:18]:
```suggestion
- Imprime a quantidade de tempo que um usuário específico está conectado em horas por dia (com total):
```

File: pages.pt_BR/common/ac.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Imprimir a quantidade de tempo que o usuário atual está conectado em horas:
+
+`ac`
+
+- Imprimir a quantidade de tempo que os usuários estão conectado em horas:
+
+`ac -p`
+
+- Imprimir a quantidade de tempo que um usuário específico está conectado em horas:
```
Comment (navarroaxel) [inline pages.pt_BR/common/ac.md:14]:
```suggestion
- Imprime a quantidade de tempo que um usuário específico está conectado em horas:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**